docs/src/app/components/pages/components/dialog.jsx

```diff
@@ -72,6 +76,12 @@ export default class DialogPage extends React.Component {
             desc: 'Set to true to have the dialog automatically open on mount.',
```

Comment:
@oliviertassinari I'll be releasing 0.13.1 shortly. This PR can go in 0.13.2. Could you please review this when you get a chance and merge it as you think fit (after 0.13.1 release though)?